### PR TITLE
Create SECURITY.md adding security policy on GH for Redisearch repo

### PR DESCRIPTION
**Pull request by adrianoamaral**
Adding the Security Policy following the Redis template https://github.com/redis/redis/security

**Describe the changes in the pull request**

A clear and concise description of what the PR is solving, including:
1. The current state briefly - Adding a security.md following the redis repo template https://github.com/redis/redis/security
3. What is the change - adding the Security Policy and the ability to users/developers report vulnerabilities in a specific tab leveraging the GH feature for it
4. Adding the outcome (changed state) - new security.MD stating the redis policy to report vulnerabilities

**Which issues this PR fixes**
1. #...
2. MOD...

**Main objects this PR modified**
1. ...
2. ...

**Mark if applicable**

- [ ] This PR introduces API changes
- [ ] This PR introduces serialization changes

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.2.x   | :white_check_mark: |
+| 7.0.x   | :white_check_mark: |
+| 6.2.x   | :white_check_mark: |
+| < 6.2   | :x:                |
```
Comment (GuyAv46) [inline SECURITY.md:17]: shouldn't this table refer to RediSearch versions?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Reporting a Vulnerability
+
+If you believe you've discovered a serious vulnerability, please contact the
+Redis core team at redis@redis.io. We will evaluate your report and if
+necessary issue a fix and an advisory. If the issue was previously undisclosed,
+we'll also mention your name in the credits.
```
Comment (GuyAv46) [inline SECURITY.md:24]: don't we want the community to report with a ticket as well?

Comment (adrianoamaral) [reply SECURITY.md:24]: It supposed, after have the policy in place, to add one button to the community report it

File: SECURITY.md (hunk 1 above)
Comment (adrianoamaral) [reply SECURITY.md]: Changes made!

**Review by GuyAv46 — APPROVED** [review]

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Redis is generally backwards compatible with very few exceptions, so we
```
Comment (LiorKogan) [inline SECURITY.md:5]: Please change "Redis" to "RediSearch"

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+recommend users to always use the latest version to experience stability,
+performance and security.
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.8.x   | :white_check_mark: |
```
Comment (LiorKogan) [inline SECURITY.md:14]: Please remove the ".x" - we only support the latest patch - trivially

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+| 2.6.x   | :white_check_mark: |
+| 2.4.x   | :white_check_mark: |
```
Comment (LiorKogan) [inline SECURITY.md:16]: 2.4 is EOL. Please remove this line and change the next one to "<2.6"

File: SECURITY.md (hunk 3 above)
Comment (adrianoamaral) [reply SECURITY.md]: Solved

File: SECURITY.md (hunk 5 above)
Comment (adrianoamaral) [reply SECURITY.md]: Solved

**Review by LiorKogan — APPROVED** [review]

**Pull request merged**